### PR TITLE
Match sticker to v1 sticker

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?
Updates v2 payment modal sticker icon. Note, there will be more stack changes both up and down stack to addess the:

- Lack of types for `style` on the sticker component
- remove additional divs
- remove tooltip

Follow up: Changes have been addressed
this is just the smallest and easiest to check change. Literally just changing the icon here.

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings
v1 on left, v2 on right
CleanShot 2022-07-23 at 12 01 14@2x

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1297** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1297" target="_blank">Graphite</a> 
    * **PR #1506** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1506" target="_blank">Graphite</a> 
      * **PR #1508** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508?utm_source=stack-comment).


Job Reference: FBdSlkLHRNUzrDzyrjTF

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508?utm_source=stack-comment).


Job Reference: oBmMxd6AzeFCh9ulYJBl

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508?utm_source=stack-comment).


Job Reference: hcVmZSl0NtpcQYTG1Iw4

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1508?utm_source=stack-comment).


Job Reference: ukIh7TJlk56j5lLaSiwO

**Pull request merged**
